src/component/feature/FeatureToggleList/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useFeatures } from '../../../hooks/api/getters/useFeatures/useFeatures';
+import useUiConfig from '../../../hooks/api/getters/useUiConfig/useUiConfig';
+import { useFeaturesFilter } from '../../../hooks/useFeaturesFilter';
+import FeatureToggleList from './FeatureToggleList';
+import { useFeaturesSort } from '../../../hooks/useFeaturesSort';
+
+export const FeatureToggleListContainer = () => {
```

Comment:
Prefer to have the file named the same as the component instead of index.tsx, it makes it easier when searching for that component in the IDE.